PlutoHelperAgent/AppDelegate.m

```diff
@@ -109,4 +109,10 @@ - (void) awakeFromNib {
     self.statusBar.highlightMode = YES;
 }
 
+- (IBAction)preferencesClicked:(id)sender {
+    if(!_preferencesWindowController) {
```

Comment:
do we need to alloc here? It's been a while since i did any of this, but can't it be done by Interface builder?

Reply:
I do not know. This worked so I used it.

Reply:
If you have ARC on that's taken care of these days.

Reply:
@blishen I think that it's always enabled now?  But does that mean that this is the right way to do it, or should it be done in IB?